ml-dsa/tests/proptests.proptest-regressions

```diff
@@ -0,0 +1,8 @@
+# Seeds for failure cases proptest has generated in the past. It is
+# automatically read and these particular cases re-run before any
+# novel cases are generated.
+#
+# It is recommended to check this file in to source control so that
+# everyone who runs the test benefits from these saved cases.
```

Comment:
I don't love having this gigantic, un-interpretable file in the repo.  Can it get pruned?  Maybe we could reset it after we merge the known issues in #895.

Reply:
It's SOP for proptest to check in these files. See description here: https://altsysrq.github.io/proptest-book/proptest/getting-started.html

> If we look at the top directory after the test fails, we’ll see a new proptest-regressions directory, which contains some files corresponding to source files containing failing test cases. These are [failure persistence](https://altsysrq.github.io/proptest-book/proptest/failure-persistence.html) files. The first thing we should do is add these to source control.

It's a log of every failure that has been found in the code so far so you automatically get regression tests for all of them.